### ISSUE TITLE
Milestone 4 Resubmission

### ISSUE THREAD

**Issue by marpierc** [airavata-courses/TeamIndra#39]
Both Karst and BR2 submissions are working, if not at the same time. As per the assignment you also need to do two things via the GitHub wiki or in a .md file:

Design and publish schema in GitHub for your metadata system.
Design and publish a description of your information management system.

**Comment by marpierc** [comment]
Post the links to these when you resubmit.  Please do this by noon Friday, May 6th.  Everyone on the team should resubmit the assignment via Canvas.

**Comment by sanketmhaiskar** [comment]
Hi Marlon,
Can we come to meet you today at the CIB ? Have some issues setting up
MySQL server on AWS on a new instance that we created.

Have a great day!!

Thanks,
Sanket Mhaiskar

On Thu, May 5, 2016 at 5:45 PM, Marlon Pierce notifications@github.com
wrote:

> Post the links to these when you resubmit. Please do this by noon Friday,
> May 6th. Everyone on the team should resubmit the assignment via Canvas.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/airavata-courses/TeamIndra/issues/39#issuecomment-217289127

## 

Thanks and Regards,
Sanket Mhaiskar

**Comment by marpierc** [comment]
Yes, I am in the CIB Multipurpose Room near the west entrance on the ground floor until 12:00 noon.

**Comment by bhandaresagar** [comment]
Hi Professor,
We have published Design of schema for your metadata system on wiki https://github.com/airavata-courses/TeamIndra/wiki/Project-Milestone-4
